### PR TITLE
INSTUI-2845 - Fix Bidirectional Context Issue 

### PR DESCRIPTION
**Pull request by Brailor**
…y,ui-view): refactor `bidirecti

Closes: INSTUI-2845

### CODE REVIEW

**Comment by matyasf** [comment]
DrawerLayout and Menu tests are failing.

File: packages/ui-drawer-layout/src/DrawerLayout/DrawerTray/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/require-default-props */
```
Comment (matyasf) [inline packages/ui-drawer-layout/src/DrawerLayout/DrawerTray/index.js:1]: Is this in the wrong place? We should not exclude whole files from linting if possible

File: packages/ui-popover/src/Popover/index.js [hunk 2]
```diff
@@ -238,7 +239,48 @@ class Popover extends Component {
...
+     */
+    onShow: PropTypes.func,
+    /**
+     * __Deprecated - use `onHideContent`__
+     */
+    onDismiss: PropTypes.func,
+    /**
+     * __Deprecated - use `onShowContent` and `onHideContent`__
+     */
+    onToggle: PropTypes.func,
```
Comment (matyasf) [inline packages/ui-popover/src/Popover/index.js:280]: Why did you add deprecated props back here?

Comment (Brailor) [reply packages/ui-popover/src/Popover/index.js]: By mistake when rebasing next into my branch, should be fixed.

File: packages/ui-drawer-layout/src/DrawerLayout/DrawerTray/index.js (hunk 1 above)
Comment (Brailor) [reply packages/ui-drawer-layout/src/DrawerLayout/DrawerTray/index.js]: Fixed in all of the files which was affected.

File: packages/ui-drawer-layout/src/DrawerLayout/DrawerTray/index.js [hunk 3]
```diff
@@ -143,11 +143,16 @@ class DrawerTray extends Component {
...
     makeStyles: PropTypes.func,
     // eslint-disable-next-line react/require-default-props
-    styles: PropTypes.object
+    styles: PropTypes.object,
+    // eslint-disable-next-line react/require-default-props
+    dir: PropTypes.oneOf(Object.values(bidirectional.DIRECTION)),
+    // eslint-disable-next-line react/require-default-props
+    rtl: PropTypes.bool,
+    // eslint-disable-next-line react/require-default-props
+    ltr: PropTypes.bool
```
Comment (ke1k0) [inline packages/ui-drawer-layout/src/DrawerLayout/DrawerTray/index.js:155]: Are rtl and ltr props needed or just dir? Also, please add them in the ui-docs-client/Properties `renderRows` method as exceptions, so they don't appear in the docs.

Comment (Brailor) [reply packages/ui-drawer-layout/src/DrawerLayout/DrawerTray/index.js]: Done

**Review by ke1k0 — APPROVED** [review]
